chainerx_cc/chainerx/native/native_device/arithmetic.cc

```diff
@@ -235,6 +235,8 @@ class NativePowerSAKernel : public PowerSAKernel {
 
 CHAINERX_NATIVE_REGISTER_KERNEL(PowerSAKernel, NativePowerSAKernel);
 
+CHAINERX_NATIVE_REGISTER_ELTWISE_BINARY_KERNEL(FmodKernel, { out = chainerx::Fmod(x1, x2); });
```

Comment:
Please add overloading to integer types like remainder:
https://github.com/chainer/chainer/pull/7675/files#diff-cc98d139a41fe00c17b77a83d1ae42dfR238